bot/code_review_bot/workflow.py

```diff
@@ -202,12 +242,12 @@ def index(self, revision, **kwargs):
                 },
             )
 
-    def find_issues(self, revision):
+    def find_issues(self, revision, group_id):
         """
         Find all issues on remote Taskcluster task group
         """
         # Load all tasks in task group
-        tasks = self.queue_service.listTaskGroup(settings.try_group_id)
+        tasks = self.queue_service.listTaskGroup(group_id)
```

Comment:
Here too, the continuationToken stuff. Probably in the try case you didn't need it as there were always just few tasks.

Reply:
Yeah, it works OK for now

Reply:
It works OK with try, but it won't work with autoland.

Reply:
Ah, you're only using it for try!

Reply:
(It's still worth fixing btw, can you file a follow-up?)